libstuff/SHTTPSManager.cpp

```diff
@@ -239,3 +240,57 @@ bool SStandaloneHTTPSManager::_onRecv(Transaction* transaction)
     transaction->response = getHTTPResponseCode(transaction->fullResponse.methodLine);
     return false;
 }
+
+list<SStandaloneHTTPSManager::Transaction*> SStandaloneHTTPSManager::_httpsSendMultiple(const string& url, vector<SData>& sendRequests)
```

Comment:
I would refactor this and `_httpsSend` so that this function calls `_httpsSend` in a loop.

I would also either pass a `list` of requests, or make this a template function that can take any collection type of requests.